site/lib/main.dart

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2025, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:jaspr/server.dart';
+import 'package:jaspr_content/jaspr_content.dart';
+import 'package:jaspr_content/theme.dart';
+import 'package:liquify/liquify.dart' show FilterRegistry;
+import 'package:path/path.dart' as path;
+
+import 'archive/archive_table.dart';
+import 'components/card.dart';
+import 'components/tabs.dart';
+import 'extensions/registry.dart';
+import 'jaspr_options.dart'; // Generated. Do not remove or edit.
+import 'layouts/doc_layout.dart';
+import 'layouts/homepage_layout.dart';
+import 'loaders/data_processor.dart';
+import 'markdown/markdown_parser.dart';
+import 'pages/custom_pages.dart';
+import 'pages/diagnostic_index.dart';
+import 'pages/lint_index.dart';
+import 'pages/robots_txt.dart';
+import 'util.dart';
+
+void main() {
+  // Initializes the server environment with the generated default options.
+  Jaspr.initializeApp(options: defaultJasprOptions);
+
+  // TODO(parlough): Eventually migrate away from
```

Comment:
nit: Convert to issue (and at least one more below).

Reply:
I've opened https://github.com/dart-lang/site-www/issues/6840 and updated the comment to point to it.